modules/admin_manual/pages/maintenance/migrating_to_kiteworks.adoc

```diff
@@ -160,14 +160,23 @@ For ease of migrating external mounts, the admin should:
 
 === Kiteworks
 
-* As a major prerequisite, the Kiteworks instance *must* be running version 8.6 or higher.
+* The following major prerequisites apply:
+
+** The Kiteworks instance *must* be running version 8.6 or higher.
+** Kiteworks requires a `Data migrator` license. +
+The respective `Include feature` checkbox must be set to enabled.
```

Comment:
lets just drop this line - end users have no understanding of the license generation.
and people generating licenses know what to do aka not our job to document this.